doc/specs/#1564 - Settings UI/spec.md

```diff
@@ -0,0 +1,163 @@
+
+## Potential Issues
+
+## Future considerations
+
+- We will have to have design reviews for all of the content pages.
+- We should have undo functionality. In a text editor, you can type `Ctrl+Z` however the settings UI is a bit more complex.
+- Once we have a marketplace for themes and extensions, this should be added to the top-level navigation.
+- As we add more features, the top-level navigation is subject to change in favor of improved usability.
+
+## Resources
+
+## Appendix 1
+
+Below is the list of all settings on their respective pages in the settings UI. The title row aligns with the navigation view on the left of the UI. Bolded headers in those columns align with top nav on the page.
+
+| General - Launch | General - Interaction | General - Rendering | Appearance - Global | Appearance - Color Schemes | Profiles - Global | Profiles - Enumerate profiles | Profiles - Add new |
+| ---------------- | --------------------- | ------------------- | ------------------- | -------------------------- | ----------------- | ----------------------------- | ------------------ |
+| Default profile (dropdown) | Copy after selection is made (checkbox) | Windows resize behavior (checkbox) | Theme (radio) | Name (text box) | **General** | **General** | **General** | **General** |
+| Launch on startup (checkbox) | Copy formatting (checkbox) | Screen redrawing (checkbox) | Show/Hide the title bar (checkbox) | Cursor color (color picker) | Command line (text box) | GUID (text box) | GUID (text box) |
+| Launch size (radio) | Word delimiters (text box) | Software rendering (checkbox) | Show terminal title in title bar (checkbox) | Selection background (color picker) | Starting directory (text box) | Command line (text box) | Command line (text box) |
+| Launch position (text box) | | | Always show tabs (checkbox) | Background (color picker) | Icon (text box) | Starting directory (text box) | Starting directory (text box) |
+| Columns on first launch (number picker) | | | Tab width mode (radio) | Foreground (color picker) | Tab title (text box) | Name (text box) | Name (text box) |
+| Rows on first launch (number picker) | | | Hide close all tabs popup (checkbox) | Black (color picker) | Scrollbar visibility (radio) | Icon (text box) | Icon (text box) |
+| Disable dynamic profiles (checkbox) | | | | Blue (color picker) | **Appearance** | Tab title (text box) | Tab title (text box) |
+| | | | | Cyan (color picker) | Font face (text box) | Scrollbar visibility (radio) | Scrollbar visibility (radio) |
+| | | | | Green (color picker) | Font size (number picker) | **Appearance** | **Appearance** |
+| | | | | Purple (color picker) | Font weight (text box) | Font face (text box) | Font face (text box) |
+| | | | | Red (color picker) | Padding (text box) | Font size (number picker) | Font size (number picker) |
+| | | | | White (color picker) | Cursor shape (radio) | Font weight (text box) | Font weight (text box) |

```

Comment:
```suggestion
| | | | | White (color picker) | Cursor shape (radio) | Font weight (dropdown) | Font weight (dropdown) |
```
Thoughts? We could just present the localized words (i.e.: light, extra-light, etc...), and if a user wants to put a number in, you should go into the json?